src/adv.py

```diff
@@ -49,3 +79,70 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+print("[n] North [e] East [s] South [w] West; [i] inventory; drop [item]; get [item] ")
+directions = input("~~~>")
+
+def reset():
+    print("[n] North [e] East [s] South [w] West; [i] inventory; drop [item]; get [item] ")
+    global directions
+    directions = input("~~~>")
+
+
+
```

Comment:
You could just put the `input` call at the top of your while loop, and eliminate the need for `reset()`